### ISSUE TITLE
Feature Request: more button images

### ISSUE THREAD

**Issue by hanjo** [mjbnz/sd-api-request#5]
First of all: thanks for this plugin - this really opens up a variety of possibilities with external systems.

Especially the option to show different images based on the status is extremely useful. Only thing I'm missing is a possibility to use more images based on the return value. Right now, you can have one picture for a match (e.g. "on") and another picture for an unmatch (anything really, but could be "off"). It would be great to have more values here, for example one could be showing 0%, 50%, 100% for a dimmable light. I image do have the possibility to have more than multiple match possibilites and one unmatch. Do you think this can be implemeted?

Thanks!

**Comment by SkyBerlin** [comment]
Yes, that would be nice! Unfortunately animated GIFs are not supported either. Maybe that can be implemented as well?
